arbnode/redislock/redis.go

```diff
@@ -192,10 +193,34 @@ func (l *SimpleRedisLock) Start(ctxin context.Context) {
 	}
 }
 
-func (l *SimpleRedisLock) StopAndWait() {
+func (l *Simple) StopAndWait() {
 	l.mutex.Lock()
 	l.stopping = true
 	l.mutex.Unlock()
 	l.Release(l.GetContext())
 	l.StopWaiter.StopAndWait()
 }
+
+func execTestPipe(pipe redis.Pipeliner, ctx context.Context) error {
```

Comment:
nit. can we remove execTestPipe, atomicTimeRead and atomicTimeWrite from seq_coordinator.go  and use these everywhere? Assuming that this still avoids the circular dependency.

Reply:
This isn't part of my pr (I haven't touched this functions even though github shows as greent). This is the code from seq_coordinator.go which I guess vscode pulled into this package as I renamed the file and package.

I can look into that as part of refactorings I'm doing.